src/pydp/algorithms/partition_selection.py

```diff
@@ -0,0 +1,16 @@
+from .._pydp import _algorithms
+
+PartitionSelectionStrategy = _algorithms.PartitionSelectionStrategy
+PartitionSelectionStrategy.__doc__ = """
+Provides a common abstraction for PartitionSelectionStrategy. Each partition
+selection strategy class has a builder with which it can be instantiated, and
```

Comment:
In this PR in Python interface there are factory functions instead of builders, so maybe something line
" has a factory function. " 
And to list somewhere the list of all factory functions

Reply:
Used `.pyi` file instead for interface.